### PR TITLE
Remove not used return values in indexing methods

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java [hunk 1]
```diff
@@ -60,23 +60,20 @@ public Indexer(String type) {
...
     @SuppressWarnings("unchecked")
-    public String performSingleRequest(T baseIndexedBean, S baseType) throws IOException, CustomResponseException {
+    public void performSingleRequest(T baseIndexedBean, S baseType) throws IOException, CustomResponseException {
         IndexRestClient restClient = initiateRestClient();
-        String response;
 
         if (method == HTTPMethods.PUT) {
             HttpEntity document = baseType.createDocument(baseIndexedBean);
-            response = String.valueOf(restClient.addDocument(document, baseIndexedBean.getId()));
+            restClient.addDocument(document, baseIndexedBean.getId());
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Why are you ignoring return value of `addDocument` method? Previous behaviour was even wrong as this method is returning a boolean value, but this is an other story.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java [hunk 2]
```diff
@@ -60,23 +60,20 @@ public Indexer(String type) {
...
         } else if (method == HTTPMethods.DELETE) {
-            response = String.valueOf(restClient.deleteDocument(baseIndexedBean.getId()));
+            restClient.deleteDocument(baseIndexedBean.getId());
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Why are you ignoring return value of `deleteDocument` method? Previous behaviour was even wrong as this method is returning a boolean value, but this is an other story.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java [hunk 3]
```diff
@@ -85,40 +82,35 @@ public String performSingleRequest(T baseIndexedBean, S baseType) throws IOExcep
...
      *            response from the server
      */
-    public String performSingleRequest(Integer beanId) throws IOException, CustomResponseException {
+    public void performSingleRequest(Integer beanId) throws IOException, CustomResponseException {
         IndexRestClient restClient = initiateRestClient();
-        String response;
 
         if (method == HTTPMethods.DELETE) {
-            response = String.valueOf(restClient.deleteDocument(beanId));
+            restClient.deleteDocument(beanId);
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Why are you ignoring return value of `deleteDocument` method? Previous behaviour was even wrong as this method is returning a boolean value, but this is an other story.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java (hunk 1 above)
Comment (Beacze) [reply Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Boolean value is only useful for testing purpose. This whole string return is useless here. We should be notified if something goes wrong by exceptions, not return values.

Comment (henning-gerhardt) [reply Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Any static code analyse tool will report this ignored return value from a method as bad programming.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java (hunk 3 above)
Comment (Beacze) [reply Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Boolean is not returned anymore.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java (hunk 2 above)
Comment (Beacze) [reply Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Boolean is not returned anymore.

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java (hunk 1 above)
Comment (Beacze) [reply Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java]: Boolean is not returned anymore.

**Pull request merged**